v6/rest/guild.ts

```diff
@@ -35,11 +48,11 @@ export interface RESTPostAPIGuildsJSONBody {
 	verification_level?: GuildVerificationLevel;
 	default_message_notifications?: GuildDefaultMessageNotifications;
 	explicit_content_filter?: GuildExplicitContentFilter;
-	roles?: APIRole[];
+	roles?: APIGuildCreateRole[];
 	channels?: APIGuildCreatePartialChannel[];
-	afk_channel_id?: string;
+	afk_channel_id?: number;
```

Comment:
Can't these be a number OR a string? That's how I understood it at least

Reply:
well, where would you get the string ID from? since the channels won't exist, although i remember trying this i think and i just got an invalid snowflake error from the API

Reply:
I mean you can generate IDs yourself, hence why I asked! If you could try it again, to confirm, that'd be awesome, and we can get this merged before v8 support 🎉

Reply:
hm, it does seem you can pass in a snowflake as the placeholder IDs too, will update the PR to reflect that later

Reply:
done, c36c7c3126e50276b448e4c0e735aea4f79dda36